ios/web_view/web_view_sources.gni

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# This file exposes the //ios/web_view embedder as a regular source_set
+
+ios_web_view_public_headers = [
```

Comment:
is this different from `ios_web_view_public_headers` in `ios/web_view/BUILD.gn`? If so it would be better to use `+=`/`-=` instead of duplicating

Reply:
The parts under our own 1 extra public header aren't different but since the original list is defined in a GN file and not GNI I couldn't access them to add into our framework so for now it was duped. If its possible to get to let me know!